server.go

```diff
@@ -710,11 +710,12 @@ func newServer(listenAddrs []net.Addr, chanDB *channeldb.DB,
 	s.missionControl, err = routing.NewMissionControl(
 		chanDB,
 		&routing.MissionControlConfig{
-			AprioriHopProbability: routingConfig.AprioriHopProbability,
-			PenaltyHalfLife:       routingConfig.PenaltyHalfLife,
-			MaxMcHistory:          routingConfig.MaxMcHistory,
-			AprioriWeight:         routingConfig.AprioriWeight,
-			SelfNode:              selfNode.PubKeyBytes,
+			AprioriHopProbability:   routingConfig.AprioriHopProbability,
+			PenaltyHalfLife:         routingConfig.PenaltyHalfLife,
+			MaxMcHistory:            routingConfig.MaxMcHistory,
+			AprioriWeight:           routingConfig.AprioriWeight,
+			SelfNode:                selfNode.PubKeyBytes,
+			MinFailureRelaxInterval: routing.DefaultMinFailureRelaxInterval,
```

Comment:
My thought was that it isn't a setting that has a lot of impact. It is merely a mitigation to bring some improvement in specific cases. It is something different than for example the payment attempt cost which has a lot of impact on path finding.

In the future we may also change the mechanism to deal with these overwrites which makes this particular parameter setting not applicable anymore. 

Another question is how many users are going to understand this parameter. You need quite deep knowledge of mission control to know what it is doing. And to infer the effect of changing it.

Because of those reasons, I think it is okay to leave it internal. But if people feel that we should expose it, I can do that too.

Reply:
I agree, we don't have to make this configurable. It is a mitigation to make a non-deterministic case (two concurrent HTLCs being failed about the same time) slightly more deterministic. 

As @joostjager mentions it is kind of a whack-a-mole to help the current, very simplistic splitting algorithm, I envision we'll handle this differently in the future.

Reply:
SGTM, we can always expose it later if need be